### ISSUE TITLE
Nullish coalescing operator

### ISSUE THREAD

**Issue by lagden** [sveltejs/eslint-plugin-svelte3#67]
For `.js` files is ok, but for `.svelte`

```js
export let disabled = options.disabled ?? false

// eslint: error
// ParseError - Unexpected token
// :( 
```

**Comment by lagden** [comment]
My `.eslintrc.js`

```js
'use strict'

const ignoreWarnings = new Set(['a11y-no-onchange'])

module.exports = {
	parserOptions: {
		ecmaVersion: 2020,
		sourceType: 'module'
	},
	env: {
		es2020: true,
		browser: true
	},
	plugins: ['html', 'svelte3'],
	extends: ['xo', 'plugin:unicorn/recommended'],
	overrides: [
		{
			files: ['**/*.svelte'],
			processor: 'svelte3/svelte3'
		}
	],
	rules: {
		indent: ['error', 'tab'],
		'linebreak-style': ['error', 'unix'],
		quotes: ['error', 'single'],
		semi: ['error', 'never'],
		'semi-spacing': [
			'error',
			{
				before: false,
				after: true
			}
		],
		'no-console': 0,
		'no-debugger': 0,
		'no-unused-expressions': [
			'error',
			{allowShortCircuit: true, allowTernary: true}
		],
		camelcase: 0,
		'capitalized-comments': 0,
		'spaced-comment': 0,
		'padding-line-between-statements': 0,
		'unicorn/filename-case': 0,
		'unicorn/prevent-abbreviations': 0,
		'unicorn/no-reduce': 0,
		// Bug do svelte lint
		'no-multiple-empty-lines': ['error', {max: 2, maxBOF: 2, maxEOF: 0}],
		// Bug no ctx.body Koa
		'require-atomic-updates': 0
	},
	settings: {
		'svelte3/ignore-warnings': w => {
			return ignoreWarnings.has(w && w.code)
		},
		'svelte3/ignore-styles': attributes => attributes.postcss
	}
}
```

**Comment by Conduitry** [comment]
Svelte itself doesn't support the nullish coalescing operator yet, but it will soon, at which point this should just work, assuming you have a new enough version of ESLint.
